manim/mobject/mobject.py

```diff
@@ -383,15 +383,15 @@ def add_to_back(self, *mobjects: "Mobject") -> "Mobject":
         :meth:`add`
 
         """
+        if self in mobjects:
+            raise ValueError("Mobject cannot contain self")
```

Comment:
maybe it will be less confusing
```suggestion
            raise ValueError("Mobjects shouldn't contain itself")
```